Corrected compile errors

DavidEGutierrez opened the pull request:
## Corrected compile errors

1. Updated title on the index.html
2. Updated PDFTron references in LICENSE
3. Updated package versions in package-lock.json
4. Update package versions, and `start` script on package,json
5. Updated README contents: links, added Apryse references in place of PDFTron, and restructured some sections.
6. removed 'build' script on package.json as it is not needed on this project. All you need is 'start' script and all steps are completed to run server with the needed resources.


## Update on removed script and typo

1. Removed mention of build script as it is not needed.
2. corrected typo: comma to period.

DavidEGutierrez committed:
Corrected compile errors

1. Updated title on the index.html
2. Updated PDFTron references in LICENSE
3. Updated package versions in package-lock.json
4. Update package versions, and `start` script on package,json
5. Updated README contents: links, added Apryse references in place of PDFTron, and restructured some sections.
6. removed 'build' script on package.json as it is not needed on this project. All you need is 'start' script and all steps are completed to run server with the needed resources.

DavidEGutierrez committed:
Update on removed script and typo

1. Removed mention of build script as it is not needed.
2. corrected typo: comma to period.

mparizeau-pdftron commented on LICENSE:
```diff
@@ -1,2 +1,5 @@
 Copyright 2020 PDFTron Systems Inc. All rights reserved.
```
Can we make this license 2024 Apryse? For example https://github.com/ApryseSDK/webviewer-ui/blob/10.12/LICENSE

mparizeau-pdftron commented on package.json:
```diff
@@ -2,10 +2,10 @@
+  "main": "build/index.js",
   "scripts": {
-    "start": "parcel index.html",
-    "build": "parcel build",
```
Do we need to get rid of the `build` script? I think the purpose is for making a "release build"

DavidEGutierrez replied:
Restored `build` script.  It seemed we needed to just showcase the app running locally and `build` had no impact on the local test. But point is taken.
 
What I did notice for my test was that the 'start` command was not loading the app on a browser. So, I added that piece of script to make it happen as seen in the previous commit: 

```
--open http://localhost:1234`
```

mparizeau-pdftron commented on styles.css:
```diff
@@ -0,0 +1,9 @@
+body{
```
Is this file use anywhere?

DavidEGutierrez replied:
The idea was to move the inline css code from html in `index.html` :
`<div id='viewer' style='width: 100%; height: 100vh; margin: 0 auto;'></div>`. I had on my test added the style tag to the index.html. 

But to avoid another change to the index.html, I am removing the css file as it has the same content.

mparizeau-pdftron commented on package.json:
```diff
@@ -2,10 +2,10 @@
   "description": "A simple JS app to demonstrate how to embed PDFTron's WebViewer",
-  "main": "index.js",
+  "source": "index.js",
+  "main": "build/index.js",
```
Doesn't seem like there is a build folder in this repo?

DavidEGutierrez replied:
The build folder is not needed in the repo as it is created from `npm run build`. 
It can be any other folder name as indicated in the `main` as the output. 

```
 "source": "index.js",
  "main": "build/index.js"
```  
  
Test by changing it to another folder name, say `test` in `"main": "test/index.js"` and it will build to that folder.

mparizeau-pdftron replied:
Ah ok I understand now. Let's just do `build/index.js` then

DavidEGutierrez committed:
Updated due to review comments from Matt Parizeau

1. Updated LICENSE to match https://github.com/ApryseSDK/webviewer-ui/blob/10.12/LICENSE
2. Restored `parcel build`, it seemed it was not necessary to run the sample project.
3. Removed styles.css as initially that was intended to hold the inline code from index.html. In the end that was not changed and will leave index.html as is.

mparizeau-pdftron commented on package.json:
```diff
@@ -2,11 +2,12 @@
   "description": "A simple JS app to demonstrate how to embed PDFTron's WebViewer",
-  "main": "index.js",
+  "source": "index.js",
+  "main": "test/index.js",
```
Ok let's just change this to `build/index.js` then. Sorry for the confusion.
